Documentation/Examples.md

```diff
@@ -71,6 +71,25 @@ label.textColor = R.color.appColors.textColor()
 
 There are some points to keep in mind when using Color palettes, see [About Colors](Colors.md)
 
+## Localized strings
+
+*Vanilla*
+```swift
+let welcomeMessage = NSLocalizedString("welcome.message", comment: "")
+let settingsTitle = NSLocalizedString("title", tableName: "Settings", comment: "")
+
+let welcomeName = String(format: NSLocalizedString("welcome.withName", comment: ""), locale: NSLocale.currentLocale(), "Alice")
```

Comment:
`String(format:arguments:)` uses the system locale, which is different from the current locale.

The [current locale](https://developer.apple.com/library/ios/documentation/Cocoa/Reference/Foundation/Classes/NSLocale_Class/index.html#//apple_ref/occ/clm/NSLocale/currentLocale):
> The logical locale for the current user. The locale is formed from the settings for the current user’s chosen system locale overlaid with any custom settings the user has specified.

Which is why it use used when formatting numbers and pluralisations.

Also this example matches the code that is generated: https://github.com/mac-cain13/R.swift/pull/188/files#diff-c67e8c23064b37ffa3ce51001930a68fR242